doc/releases/changelog-0.39.0.md

```diff
@@ -4,220 +4,384 @@
+
+* The `qml.FABLE` template now returns the correct value when jit is enabled.
+  [(#6263)](https://github.com/PennyLaneAI/pennylane/pull/6263)
+
+* `qml.QutritBasisStatePreparation` is now jit compatible.
+  [(#6308)](https://github.com/PennyLaneAI/pennylane/pull/6308)
+
+* The `SampleMP.process_samples` method has been updated to support using JAX tracers for samples, allowing 
+  compatiblity with qjit workflows.
+  [(#6211)](https://github.com/PennyLaneAI/pennylane/pull/6211)
+
 <h4>Capturing and representing hybrid programs</h4>
 
-* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised in `JAX 0.4.30+`
-  when providing JAX tracers as input to `qml.wires.Wires`.
+* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised 
+  in `JAX 0.4.30+` when providing JAX tracers as input to `qml.wires.Wires`.
   [(#6312)](https://github.com/PennyLaneAI/pennylane/pull/6312)
 
-* Differentiation of hybrid programs via `qml.grad` and `qml.jacobian` can now be captured
-  into plxpr. When evaluating a captured `qml.grad` (`qml.jacobian`) instruction, it will
-  dispatch to `jax.grad` (`jax.jacobian`), which differs from the Autograd implementation
-  without capture. Pytree inputs and outputs are supported.
+* A new function called `qml.capture.make_plxpr` has been added to take a function and create a `Callable` 
+  that, when called, will return a PLxPR representation of the input function.
+  [(#6326)](https://github.com/PennyLaneAI/pennylane/pull/6326)
+
+* Differentiation of hybrid programs via `qml.grad` and `qml.jacobian` can now be captured with PLxPR. 
+  When evaluating a captured `qml.grad` (`qml.jacobian`) instruction, it will dispatch to `jax.grad` 
+  (`jax.jacobian`), which differs from the Autograd implementation without capture. Pytree inputs and 
+  outputs are supported.
   [(#6120)](https://github.com/PennyLaneAI/pennylane/pull/6120)
   [(#6127)](https://github.com/PennyLaneAI/pennylane/pull/6127)
   [(#6134)](https://github.com/PennyLaneAI/pennylane/pull/6134)
 
-* Improve unit testing for capturing of nested control flows.
+* Unit testing for capturing nested control flow has been improved.
   [(#6111)](https://github.com/PennyLaneAI/pennylane/pull/6111)
 
-* Some custom primitives for the capture project can now be imported via
-  `from pennylane.capture.primitives import *`.
+* Some custom primitives for the capture project can now be imported via `from pennylane.capture.primitives import *`.
   [(#6129)](https://github.com/PennyLaneAI/pennylane/pull/6129)
 
-* All higher order primitives now use `jax.core.Jaxpr` as metadata instead of sometimes
-  using `jax.core.ClosedJaxpr` and sometimes using `jax.core.Jaxpr`.
+* All higher order primitives now use `jax.core.Jaxpr` as metadata instead of sometimes using `jax.core.ClosedJaxpr` 
+  or `jax.core.Jaxpr`.
   [(#6319)](https://github.com/PennyLaneAI/pennylane/pull/6319)
```

Comment:
These seem like implementation details that don't necessarily belong in improvements 🤔

Reply:
The improvements section has been our "catch-all" section. It's comprises newer features that didn't quite make the cut for being top-level highlights, all the way down to minor tweaks to the code base. We can consider splitting this up? Maybe just another section called "Under the hood fixes" or something like that?

Reply:
We also have the "Other improvements" section, which is the _true_ catch-all section for all disenfranchised and abstract features haha